Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -634,6 +636,33 @@ public override void Dispose()
             Log.Debug("Store disposed.");
         }
 
+        private string ResolvePluginPath(string path) =>
+            Uri.IsWellFormedUriString(path, UriKind.Absolute)
+                ? DownloadPlugin(path).Result
+                : path;
+
+        private async Task<string> DownloadPlugin(string url)
+        {
+            var path = Path.Combine(Environment.CurrentDirectory, "plugins");
+            Directory.CreateDirectory(path);
+            var hashed = url.GetHashCode().ToString();
+            var logger = Log.ForContext("LibplanetNodeService", hashed);
+            using var httpClient = new HttpClient();
+            var downloadPath = Path.Join(path, hashed + ".zip");
+            var extractPath = Path.Join(path, hashed);
+            Directory.CreateDirectory(extractPath);
```

Comment:
I think this line have to be removed.
https://learn.microsoft.com/en-us/dotnet/api/system.io.compression.zipfile.extracttodirectory?view=net-8.0

Reply:
https://github.com/planetarium/NineChronicles.Headless/commit/24a5c2c00b43ea46849625d3945e9e7f79190b91 amended!